README.md

```diff
@@ -1,1134 +1,233 @@
 
-- Play 2.5.4+
-- Swagger (OpenAPI) 2.0
-
-## Build Status
+We built API-First-Hand for use with [Play Framework](http://www.playframework.com/), but we would like to extend it for use with **Akka HTTP**. [Get in touch](https://github.com/zalando/api-first-hand/blob/master/CONTRIBUTING.md) if you'd like to help make that possible.
 
-This plugin should be enabled using the [api-first-hand](http://www.typesafe.com/activator/template/api-first-hand) activator template 
-as the version in this repository is under active development. The status of this software is beta, 
-an end-to-end functional release intended to demonstrate the possibility to generate following from a Swagger specification:
+### Plugin Features 
 
-- Play route files
-- Generators of random test data
-- Wrappers for Play route files to convert semantics from http-related to domain-related (controller_base)
-- Skeletons for the domain-driven controller implementation
+Api-First-Hand supports round-trip regeneration and compilation of:
+- Play route definitions (managed)
+- Swagger domain model definitions and parameters onto Scala case classes (managed)
+- Swagger domain model constraints onto Play validations (managed)
+- Generators for random test data and parameter values (managed)
+- Unit tests for invalid and valid parameter sets // validating your service at the API boundary (managed)
+- Swagger path definitions onto skeletons for Play controller implementations (unmanaged)
+- Skeletons for domain-driven controller implementation and customized deserializers
+- Wrappers for Play route files to convert semantics from HTTP-related to domain-related (controller_base)
 - Model classes and validation rules
-- Unit tests for invalid and valid parameter sets
-- Security extractors (if needed)
-- Skeletons for custom deserializers (if needed)
+- Security extractors (manual generation and compilation)
+- Unmarshallers for custom content types (manual generation and compilation)
 
-We benefit from community feedback. All comments are welcome!
+Managed means "managed by sbt"—and this means that you don't have to control or change the code as you make your REST service. The security extractors and unmarshallers are available through manual generation and compilation, and supported if A) No security extractor or unmarshaller with the same name already exists; B) The developer issues the `playScalaSecurity` or `playScalaMarshallers` sbt command.
```

Comment:
@slavaschmidt I will leave this part until this part officially becomes obsolete.